packages/integration-sdk-cli/src/commands/document.ts

```diff
@@ -19,20 +19,20 @@ const J1_DOCUMENTATION_MARKER_START =
 const J1_DOCUMENTATION_MARKER_END = '<!-- {J1_DOCUMENTATION_MARKER_END} -->';
 
 interface DocumentCommandArgs extends TypesCommandArgs {
-  documentationFilePath: string;
+  outputFile: string;
 }
 
 export function document() {
   return createCommand('document')
     .description('Generates documentation for all steps')
     .option(
       '-p, --project-path <directory>',
-      'Absolute file path to the integration project directory. Defaults to the current working directory.',
+      'Absolute path to the integration project directory. Defaults to the current working directory.',
       process.cwd(),
     )
     .option(
-      '-f, --documentation-file-path <path>',
-      'Absolute file path to the Markdown file that should be created/updated. Defaults to {CWD}/docs/jupiterone.md.',
+      '-o, --output-file <path>',
```

Comment:
Ya this is a great change. Technically a breaking change, so we'll want to major this.